crates/utils/src/parser.rs

```diff
@@ -162,6 +162,24 @@ pub fn parse_state(s: &str) -> Result<InitialClusterState, ConfigParseError> {
     }
 }
 
+/// Parse `LOG_PATH` from string
+/// # Errors
+/// Return error when parsing the given string to `PathBuf` failed
+#[inline]
+pub fn parse_log_file(s: &str) -> Result<Option<PathBuf>, ConfigParseError> {
+    if s.is_empty() {
+        return Ok(None);
+    }
+    let path = PathBuf::from(s);
+    if path.exists() {
+        Ok(Some(path))
+    } else {
+        Err(ConfigParseError::InvalidValue(format!(
+            "Log file path is illegal: {s}"
+        )))
+    }
```

Comment:
Why do you check if the log file exists? I think we should validate the file path and ensure it is a valid Linux file system path, rather than just checking if it exists.